.sync.yml

```diff
@@ -7,3 +7,5 @@ spec/spec_helper.rb:
 .gitignore:
   paths:
     - spec/fixtures/tmpdir/
+Rakefile:
+  puppet_strings_patterns: 'lib/puppet/functions/extlib/*.rb'
```

Comment:
What would be the correct syntax of specifying two patterns?

If I go with something like
```
puppet_strings_patterns: 'lib/puppet/functions/extlib/*.rb,functions/*.pp'
```
I get nothing generated at all.

Reply:
https://github.com/puppetlabs/puppet-strings/blob/a2deef465a58104ed94abec668a599a29303023b/lib/puppet-strings/tasks/generate.rb#L8 suggests whitespce

Reply:
I think it needs to be `puppet_strings_patterns: 'lib/puppet/functions/extlib/*.rb functions/**/*.pp'`  Note the space instead of comma.